### ISSUE TITLE
crf_layer cost过大，出现浮点异常

### ISSUE THREAD

**Issue by zhuantouer** [PaddlePaddle/book#446]
这个角色标注的例子https://github.com/PaddlePaddle/book/blob/develop/07.label_semantic_roles/train.py
用的batch_size=10，如果把batch_size设置较大的话（256，512），就会出现浮点异常。设置coeff参数为0.3也会出现同样的问题。
想请教下 @pengli09
这个有什么优化方法么，batch_size设置过小的话，训练太慢了。

**Comment by zhuantouer** [comment]
@pengli09 您好，再请教您个问题，https://github.com/PaddlePaddle/book/blob/develop/07.label_semantic_roles/train.py
这个用feature_out作为特征接了crf。某个词W，它有个tag（B-Loc），这个词的lstm输出可以接个全连接，softmax，做成个分类问题，如果一共5个tag，算个交叉熵也可以做tag任务。
请问这个方法怎样实现呢，feature_out是什么维度的数据，可以实现我说的这种方法么？

**Comment by shanyi15** [comment]
您好，此issue在近一个月内暂无更新，我们将于今天内关闭。若在关闭后您仍需跟进提问，可重新开启此问题，我们将在24小时内回复您。因关闭带来的不便我们深表歉意，请您谅解~感谢您对PaddlePaddle的支持! 
 Hello, this issue has not been updated in the past month. we will close it today for the sake of other user’s experience. If you still need to follow up on this question after closing, please feel free to reopen it. In that case, we will get back to you within 24 hours. We apologize for the inconvenience caused by the closure and thank you so much for your support of PaddlePaddle Group!
